pages/views/examples/related-target.njk

```diff
@@ -0,0 +1,47 @@
+---
+layout: content
+title: Related Toggleables
+seoTitle: Related toggleable target example based on ESLMixinElement
```

Comment:
We can try to replace `related` with synonyms. Because the word is already used very often on the page. (for SEO reasons)

Excuse me for the silly cavils.

Reply:
Yes, thank you. I will fix it in the next pr related to this feature